scripts/vcf_site_level_filtering_wdl/JointVcfFiltering.wdl

```diff
@@ -156,12 +170,20 @@ task ExtractVariantAnnotations {
 
 		Int memory_mb = 28000
 		Int command_mem = memory_mb - 1000
+		File? monitoring_script
 	}
 	Int disk_size = ceil(size(input_vcf, "GB") + size(input_vcf_index, "GB") + 100)
 
+
 	command {
 		set -e
 
+		# Create a zero-size monitoring log file so it exists even if we don't pass a monitoring script
```

Comment:
hmm an optional output seems more clear and symmetric with the optional input

Reply:
Yeah, I'm going to have another go at that. Ran into issues with flatten, but I think optionals would be better.